src/main/java/pl/jkap/sozzt/contract/domain/ContractData.java

```diff
@@ -0,0 +1,20 @@
+package pl.jkap.sozzt.contract.domain;
+
+import lombok.Builder;
+import lombok.Getter;
+import java.time.LocalDateTime;
+
+@Builder
+@Getter
+class ContractData {
+
+    private Long id;
+    private String invoiceNumber;
+    private String location;
+    private String executive;
+    private LocalDateTime created;
```

Comment:
isn't the last updated data important for your process, maybe also who did last update too?